app/services/file_reconciliation.py

```
@@ -42,17 +48,37 @@ def reconcile_missing_symlinks(path: MonitoredPath, db: Session) -> Dict:
+            return loc.operation_mode == OperationType.MOVE and path.operation_type == OperationType.SYMLINK
+
+        symlink_location = next(
+            (loc for loc in path.storage_locations if _is_symlink_location(loc, path)),
+            None,
+        )
+
+        # Only reconcile symlinks when a local SYMLINK backend location exists.
+        if symlink_location is None:
             logger.debug(
-                f"Skipping symlink reconciliation for {path.name} - operation type is {path.operation_type.value}"
+                f"Skipping symlink reconciliation for {path.name} - no local SYMLINK location"
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)